### DISCUSSION TITLE
Why is sealing not supported for sparse writable layers?

### DISCUSSION THREAD

**Discussion by gorbak25** [containerd/overlaybd#216] [General]
I wonder whether this may be done or there is a fundamental limitation.

**Comment by BigVan** [comment]
In theory, 'close_seal' method can be implemented in LSMTSparseFile.
And people should commit the sealed sparse layer before uploading it to the registry. // When we use buildkit to create overlaybd image, there will be similar processing

The relevant code will be released later.

**Comment by BigVan** [comment]
https://github.com/containerd/overlaybd/pull/217

**Reply by gorbak25** [reply]
Thank you!
